amaranth/back/rtlil.py

```diff
@@ -1070,9 +1080,9 @@ def emit_read_port(self, cell_idx, cell):
             "WIDTH": cell.width,
             "TRANSPARENCY_MASK": _ast.Const(transparency_mask, memory_info.num_write_ports),
             "COLLISION_X_MASK": _ast.Const(0, memory_info.num_write_ports),
-            "ARST_VALUE": _ast.Const(0, cell.width),
-            "SRST_VALUE": _ast.Const(0, cell.width),
-            "INIT_VALUE": _ast.Const(0, cell.width),
+            "ARST_VALUE": Undef(cell.width),
```

Comment:
This is profoundly cursed. Could you add a comment explaining why, please? The rationale is quite invovled.

Reply:
Done.